index.html

```diff
@@ -1050,8 +1053,23 @@ <h3>Processing Model</h3>
 <li>Record the initiator of the resource in
 <a>initiatorType</a>.</li>
 <li>Record the <a data-cite="HTML#resolve-a-url">resolved URL</a>
-of the requested resource in <a>name</a>. If there is an
-<a data-cite=
+  of the requested resource in <a>name</a>.</li>
+<li><em>Fetch start:</em> <dfn data-lt="step-fetch-start">Immediately before a
+user agent starts the <a data-cite="FETCH#concept-fetch" data-lt=
+'fetch'>fetching process</a></dfn>, record the <a>current time</a>
+as <a>fetchStart</a>. Let <a>domainLookupStart</a>,
+<a>domainLookupEnd</a>, <a>connectStart</a> and <a>connectEnd</a>
+be the same value as <a>fetchStart</a>.</li>
+<li><dfn data-lt="step-collection-start">If the user agent is to
+reuse the data from another existing or completed <a data-cite=
+"FETCH#concept-fetch">fetch</a> initiated from the <a>current
+document</a></dfn>, abort the remaining steps.</li>
+<li>If the resource's <a>Request</a>'s
+<a data-cite="Fetch#concept-request-destination">destination</a> equals to
+"document" and the resource fails the <a>timing allow check</a> algorithm, the
+user agent MUST set <a>workerStart</a> to zero and go to the step labeled
+<a href="#dfn-step-response-end">response end</a>.</li>
```

Comment:
Why does this jump to response end step? In this case when would redirectStart and friends be set?

Reply:
You're right, changed the logic to merge the conditionals that lead to jumping to the response end.